docs/upgrade.html

```diff
@@ -74,24 +74,24 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
         <li>
             The Next Generation of the Consumer Rebalance Protocol (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol">KIP-848</a>)
             is now Generally Available (GA) in Apache Kafka 4.0. The protocol is automatically enabled on the server when the upgrade to 4.0 is finalized.
-            Note that once the new protocol is used by consumer groups, the cluster can only downgrade to version 3.4.1 or newer.
-            Check <a href="/{{version}}/documentation.html#consumer_rebalance_protocol">here</a> for details.
+            Note that once the new protocol is used by consumer groups, the cluster can only be downgraded to version 3.4.1 or newer.
+            For more information check <a href="/{{version}}/documentation.html#consumer_rebalance_protocol">here</a>.
         </li>
         <li>
-            Transactions Server Side Defense (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-890%3A+Transactions+Server-Side+Defense">KIP-890</a>)
+            Transactions Server-Side Defense (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-890%3A+Transactions+Server-Side+Defense">KIP-890</a>)
             brings a strengthened transactional protocol to Apache Kafka 4.0. The new and improved transactional protocol is enabled when the upgrade to 4.0 is finalized.
             When using 4.0 producer clients, the producer epoch is bumped on every transaction to ensure every transaction includes the intended messages and duplicates are not
-            written as part of the next transaction. Downgrading the protocol is safe. For more information check <a href="/{{version}}/documentation.html#transaction_protocol">here</a>
+            written as part of the next transaction. Downgrading the protocol is safe. For more information check <a href="/{{version}}/documentation.html#transaction_protocol">here</a>.
         </li>
         <li>
             Eligible Leader Replicas (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-966%3A+Eligible+Leader+Replicas">KIP-966 Part 1</a>)
```

Comment:
ditto: Could you also update the KIP link to `https://cwiki.apache.org/confluence/x/mpOzDw`? If the KIP title changes in the future, the current URL may become invalid.